terraform/node_resource_abstract.go

```diff
@@ -43,6 +43,10 @@ type GraphNodeResourceInstance interface {
 // operations. It registers all the interfaces for a resource that common
 // across multiple operation types.
 type NodeAbstractResource struct {
+	//FIXME: AbstractResources are no longer absolute, because modules are not expanded.
```

Comment:
@jbardin I think we can delete this comment, or should we leave it for Future Us?

Reply:
Nope, this still needs to be done. The Abstract resource is still using a shimmed `addrs.AbsResource`